### PR TITLE
Add pull request template

### LINKED ISSUE #30: Create a PR template
This is helpful to ensure that PRs are well-described and easy to review. Feel free to steal from https://github.com/bevyengine/bevy/blob/main/.github/pull_request_template.md

### PR DESCRIPTION
**Pull request by TimJentzsch**
Closes #30.

This adds a PR template to make PRs more consistent and potentially easier to review.

I copied over the [PR template from bevy](https://github.com/bevyengine/bevy/blob/main/.github/pull_request_template.md) without any changes, please let me know if I should make adjustments :)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- What changed as a result of this PR?
+- If applicable, organize changes under "Added", "Changed", or "Fixed" sub-headings
+- Stick to one or two sentences. If more detail is needed for a particular change, consider adding it to the "Solution" section
+  - If you can't summarize the work, your change may be unreasonably large / unrelated. Consider splitting your PR to make it easier to review and merge!
+
+## Migration Guide
+
+> This section is optional. If there are no breaking changes, you can delete this section.
+
+- If this PR is a breaking change (relative to the last release of Bevy), describe how a user might need to migrate their code to support these changes
```
Comment (alice-i-cecile) [inline .github/pull_request_template.md:25]:
```suggestion
- If this PR is a breaking change (relative to the last release of this crate), describe how a user might need to migrate their code to support these changes
```

Comment (TimJentzsch) [reply .github/pull_request_template.md]: Good catch!

**Comment by alice-i-cecile** [comment]
@TimJentzsch can you add a rule that ignores (or increases to like 160 characters) the mdlint line limit? This is a dumb and noisy requirement in the modern era IMO.

**Comment by TimJentzsch** [comment]
Sure, should this be in this PR or in another one? Maybe a good fit for the one fixing the existing lint issues?

Looks like we only have to uncomment these lines (or set it to `false`):
https://github.com/DioxusLabs/stretch/blob/b41c53783bd5e78bc1b773e43350933a56aeb669/.github/linters/.markdown-lint.yml#L24-L25

**Comment by alice-i-cecile** [comment]
> Sure, should this be in this PR or in another one? Maybe a good fit for the one fixing the existing lint issues?

I'm fine with either. Let's do it here, and get this unblocked ASAP.

**Comment by alice-i-cecile** [comment]
> Cannot read or parse config file '/github/workspace/.github/linters/.markdown-lint.yml': Unable to parse '/github/workspace/.github/linters/.markdown-lint.yml'; Parser 0: Unexpected token M in JSON at position 0; Parser 1: expected a single document in the stream, but found more

Looks like the formatting is off on our config :(

**Comment by TimJentzsch** [comment]
Ok turns out the `---` does NOT separate the "actual" rules from the examples :D
Hopefully it works now

**Pull request merged**